build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-  val kotlinVersion = "1.3.72"
+  val kotlinVersion = "1.4.0"
```

Comment:
Unfortunately the quality plugin isn't compatible with Kotlin 1.4 (due to a Ktlint issue) yet.